src/index.js

```diff
@@ -24,7 +25,7 @@ $(() => {
   createModal()
 
   const $form = $('form')
-  const $checkBoxRequirePopper = $('.require-popper')
+  const $checkBoxRequirePopper = $(document.querySelector('.require-popper'))
```

Comment:
What's this for?

Reply:
Just a matter of perf, that's always better to query our elements ourselves instead of using jQuery

Reply:
If we're going to optimise this here, we better do this everywhere. But this is out of the scope of this PR I think.